tests/eos-test-utils.h

```diff
@@ -0,0 +1,237 @@
+/* -*- mode: C; c-file-style: "gnu"; indent-tabs-mode: nil; -*-
+ *
+ * Copyright 2016 Kinvolk GmbH
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the
+ * Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+ * Boston, MA 02111-1307, USA.
+ *
+ * Author: Krzesimir Nowak <krzesimir@kinvolk.io>
+ */
+
+#pragma once
+
+#include "spawn-utils.h"
+
+#include <eos-refcounted.h>
+
+#include <gio/gio.h>
+
+G_BEGIN_DECLS
+
+typedef struct {
+  GTestDBus *dbus;
+  GFile *tmpdir;
+} EosUpdaterFixture;
+
+void eos_updater_fixture_setup (EosUpdaterFixture *fixture,
+                                gconstpointer user_data);
+
+void eos_updater_fixture_teardown (EosUpdaterFixture *fixture,
+                                   gconstpointer user_data);
+
+#define eos_test_add(testpath, tdata, ftest) g_test_add (testpath, EosUpdaterFixture, tdata, eos_updater_fixture_setup, ftest, eos_updater_fixture_teardown)
+
+gchar *get_keyid (void);
+
+static const gchar *const default_vendor = "VENDOR";
+static const gchar *const default_product = "PRODUCT";
+static const gchar *const default_ref = "REF";
+static const gchar *const default_ostree_path = "OSTREE/PATH";
+static const gchar *const default_remote_name = "REMOTE";
```

Comment:
I think these need to be `extern const gchar *` - putting them as static in the same header file will lead to symbol duplication for every `.c` that includes this header.